snapcraft/internal/project_loader/_config.py

```diff
@@ -255,6 +256,16 @@ def _get_required_package_repositories(self) -> List[PackageRepository]:
 
         return package_repos
 
+    def _verify_all_key_assets_utilized(
```

Comment:
You're checking if all keys are installed on our key manager, but we can't tell if they're actually used to validate the packages we install, is that correct? Perhaps we could reflect that in the function name (`_verify_all_key_assets_installed` instead of `_verify_all_key_assets_utilized`)?

Reply:
interesting point.  Though the intent is to determine that all key assets are utilized, not installed.  Though it does that be checking that they are all installed. :thinking:

Reply:
Fixed